src/devices/cpu/m6502/m6502_vtscr.cpp

```diff
@@ -0,0 +1,106 @@
+// license:BSD-3-Clause
+// copyright-holders:David Shah
+/***************************************************************************
+
+    m6502_vtscr.cpp
```

Comment:
Put that in mame/machine/* or directly in nes_vt.* please.  No sane reason to have it in devices/cpu/m6502